tests/ha/ctdb.pm

```diff
@@ -53,6 +54,7 @@ sub run {
 
         # Get smb conf file from the openQA server
         assert_script_run "curl -f -v " . autoinst_url . "/data/ha/smb.conf -o $ctdb_cfg";
+        assert_script_run "sed -i -e 's|%CTDB_SOCKET%|$ctdb_socket|' $ctdb_cfg";
```

Comment:
I think that you can now use `file_content_replace` function to do this.

**Note:** we will have to change all `sed` commands we previously used by the function in the future